tests/test_multiple_in_and_out_port_components.py

```diff
@@ -88,9 +97,12 @@ def test_heat_pump(self):
         sec_heat = results["GenericConversion_3d3f.Secondary_heat"]
         power_elec = results["GenericConversion_3d3f.Power_elec"]
 
-        # TODO: check if the primary source utilisisation is maximised and secondary minimised
-        # I think it is badly scaled or some scaling issues in constraints. (possible in combination
-        # with disconnected)
+        # Check that only the minimum velocity is flowing through the secondary source.
+        np.testing.assert_allclose(
+            results["ResidualHeatSource_aec9.Q"] / (np.pi * (0.3127 / 2.0) ** 2),
```

Comment:
assign (np.pi * (0.3127 / 2.0) ** 2) to a variable describing what it is, namely cross sectional area

Reply:
ok